CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 
-0.06
+
+RestInParse 0.07
+===
+
+- Implements `Parse.Query.count()`
+- New criteria `Parse.Query.isTrue()` and `Parse.Query.isNotTrue()` which search for either `false` `null` or `undefined`
+- New criteria `notEqualToInt()`, `notEqualToPtr`, `notEqualToString`
```

Comment:
Just a random idea: Jasmine has solved negations like this:
Parse.Query.isTrue()
Parse.Query.not.isTrue()
Parse.Query.equalToInt()
Parse.Query.not.equalToInt()
etc..
Maybe this is easier to implement and to use?